src/platform/packages/shared/kbn-crypto/src/pkcs12.ts

```diff
@@ -31,7 +31,7 @@ export interface Pkcs12ReadResult {
  *
  * Any other PKCS12 bags are ignored.
  *
- * @privateRemarks
+ * @internalRemarks
```

Comment:
```suggestion
 * @privateRemarks
```

This was correct https://tsdoc.org/pages/tags/privateremarks/